src/fcntl.rs

```diff
@@ -69,6 +71,8 @@ pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
             F_ADD_SEALS(flag) => libc::fcntl(fd, ffi::F_ADD_SEALS, flag.bits()),
             #[cfg(target_os = "linux")]
             F_GET_SEALS => libc::fcntl(fd, ffi::F_GET_SEALS),
+            #[cfg(target_os = "darwin")]
+            F_FULLFSYNC => libc::fcntl(fd, ffi::F_FULLFSYNC),
```

Comment:
Darwin is not a valid `target_os`. You have to use either `macos`, `ios` or both, depending on where that flag exists. This is also, why this line does not cause our apple build to fail. Otherwise it would complain about an unkown identifier `ffi::F_FULLFSYNC` (see the `ffi` module definition further up in the file).